src/dom/dom.doc.js

```diff
@@ -1,3 +1,4 @@
+/* eslint valid-jsdoc: 2 */
```

Comment:
these are included in .doc.js files so that we are warned in the jsdoc is invalid.